.storybook/config.js

```diff
@@ -12,8 +13,10 @@ function loadStories() {
 addDecorator(story => (
   /* eslint-disable react/jsx-filename-extension */
   <Fragment>
-    <GlobalStyle />
-    {story()}
+    <ServiceContextProvider service="news">
```

Comment:
I dont think service context providers should the set globally (espesh not specifically to news). If we need a `ServiceContextProvider` around a story, it should be done at the story level, or make a new Storybook decorator, like AmpDecorator, that can be added to a single story.